lib/spack/spack/package.py

```diff
@@ -380,6 +380,22 @@ def __init__(self, spec):
         if not hasattr(self, 'list_depth'):
             self.list_depth = 1
 
+        # Set default licensing information
+        if not hasattr(self, 'license_required'):
+            self.license_required = False
+
+        if not hasattr(self, 'license_comment'):
+            self.license_comment = '#'
+
+        if not hasattr(self, 'license_files'):
+            self.license_files = []
+
+        if not hasattr(self, 'license_vars'):
+            self.license_vars = []
+
+        if not hasattr(self, 'license_url'):
+            self.license_url = None
```

Comment:
I was just following what was already there. But I was pretty sure the if-statement part wasn't necessary because it would never have that attribute upon initialization and would always get overwritten later. I'll remove the if-statements for these and the list_url/list_depth so that they always get set.